test/collections/HashDictionaryTest.ooc

```diff
@@ -159,16 +159,16 @@ HashDictionaryTest: class extends Fixture {
 			dictionary add("First", 1)
 			dictionary add("Second", 2)
 			dictionary add("Third", 3)
-			expect(dictionary count, is equal to(3))
+			expect(dictionary count as Int, is equal to(3))
 			expect(dictionary empty, is false)
 			expect(dictionary contains("Second"), is true)
 			dictionary remove("Second")
-			expect(dictionary count, is equal to(2))
+			expect(dictionary count as Int, is equal to(2))
 			expect(dictionary empty, is false)
 			expect(dictionary contains("Second"), is false)
 			dictionary remove("Third")
 			dictionary remove("First")
-			expect(dictionary count, is equal to(0))
+			expect(dictionary count as Int, is equal to(0))
```

Comment:
`count` is now of type `SizeT`, which cannot be compared with `Int` easily since `rock` would rather cast it to `Float`........